src/lib.rs

```diff
@@ -981,6 +987,7 @@ macro_rules! impl_integer_for_usize {
 
             #[test]
             fn test_is_multiple_of() {
+                assert!((0 as $T).is_multiple_of(&(0 as $T)));
                 assert!((6 as $T).is_multiple_of(&(6 as $T)));
                 assert!((6 as $T).is_multiple_of(&(3 as $T)));
                 assert!((6 as $T).is_multiple_of(&(1 as $T)));
```

Comment:
Hmm, these tests have no negatives -- we could pass with `return true`. Could I get you to add a few variants, including a false test with RHS 0?

Reply:
Done!